test/users.test.js

```diff
@@ -0,0 +1,73 @@
+const request = require('supertest');
+const dictum = require('dictum.js');
+
+const app = require('../app');
+
+const controller = request(app);
+
+describe('POST /users', () => {
+  test('Successful test when registering user', () =>
+    controller
+      .post('/users')
+      .send({
+        firstName: 'John',
+        lastName: 'Katzenbach',
+        email: 'john.katz@wolox.co',
+        password: '12345678'
+      })
+      .then(response => {
+        expect(response.statusCode).toBe(201);
+        return dictum.chai(response, 'Successful test creating user');
```

Comment:
This is unnecessary. In each test, you must call `done()` in the last line test. Also add `dictum.chai(response)` for only one successful case.
Check in all test and refactor this.